IdentityCore/src/controllers/MSIDSilentController.m

```diff
@@ -90,6 +90,9 @@ - (void)acquireToken:(nonnull MSIDRequestCompletionBlock)completionBlock
 
     MSIDSilentTokenRequest *silentRequest = [self.tokenRequestProvider silentTokenRequestWithParameters:self.requestParameters
                                                                                            forceRefresh:self.forceRefresh];
+#if TARGET_OS_OSX
+    silentRequest.externalCacheSeeder = self.externalCacheSeeder;
```

Comment:
nit: can this instead be set in request directly? e.g. Token request provider can set it on requests directly when necessary? Trying to avoid having to pass it around everywhere and have all the ifdefs...

Reply:
done